app/serializers/api/admin/customer_serializer.rb

```diff
@@ -18,4 +19,9 @@ def tags
       tag_rule_map || { text: tag, rules: nil }
     end
   end
+
+  def default_card_present?
+    return unless object.user
```

Comment:
shouldn't always a customer be associated to a user?

EDIT now I recall having to manually update customer records to associated them to the appropriate user id. So it is not enforced.

```ruby
  create_table "customers", :force => true do |t|
    t.string   "email",           :null => false
    t.integer  "enterprise_id",   :null => false
    t.string   "code"
    t.integer  "user_id"
    ...
  end
```

☝️ `user_id` is nullable 🔴

Reply:
See comment on #2274, we need user_id to be nullable, this is from Spree, not something that we set up

Reply:
:sob: